app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, old_file, new_file = command.split()
+    if command == "cp":
+        with open(old_file, "r") as file_in,\
+             open(new_file, "w") as file_out:
+            for line in file_in:
+                file_out.write(line)
```

Comment:
How do you handle this?
![Screenshot from 2023-09-10 13-23-22](https://github.com/mate-academy/py-copy-file/assets/122517203/3841babf-72bb-4483-b604-dc62b292bec9)